websites/A/AnimeX/dist/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+  "author": {
+    "name": "NotMarek",
+    "id": "206299359007080450"
+  },
+  "service": "AnimeX",
+  "description": {
+    "en": "AnimeX is a free anime streaming site without ads.",
+    "cs": "AnimeX je bezplatná stránka pro streamování anime bez reklam."
```

Comment:
cs_CZ

Reply:
Deepscan denied that

Reply:
deepscan can't deny that.

Reply:
Oh sorry I had cs-CZ I will make a commit tommorow

Reply:
ah yea the - is blocked since its _ thats why it was denied.